pallets/xcmp-queue/src/lib.rs

```diff
@@ -220,6 +249,10 @@ pub mod pallet {
 	/// available free overweight index.
 	#[pallet::storage]
 	pub(super) type OverweightCount<T: Config> = StorageValue<_, OverweightIndex, ValueQuery>;
+
+	/// Whether or not the XCMP queue is active in executing incoming XCMs or not.
+	#[pallet::storage]
+	pub(super) type QueueActive<T: Config> = StorageValue<_, bool, ValueQuery>;
```

Comment:
this will be default to false so we either need a migration + genesis config to make this true, or make it `PauseQueue` or something that default value means standard behaviour.

Reply:
That was what I was wondering as well, glad that you caught it!